include/flatbuffers/flexbuffers.h

```diff
@@ -218,7 +218,8 @@ class String : public Sized {
 
   size_t length() const { return size(); }
   const char *c_str() const { return reinterpret_cast<const char *>(data_); }
-
+  std::string str() const { return c_str(); }
```

Comment:
this change seems unrelated to the other bits of the PR.  i don't know how the maintainers approach this, but in general, i believe a PR should have one logical change.

you can also avoid the overhead of the implicit strlen: ```return std::string(c_str(), length());```